pygeoapi/linked_data.py

```diff
@@ -173,104 +175,146 @@ def jsonldify_collection(cls, collection, locale_):
 
 def geojson2geojsonld(config, data, dataset, identifier=None, id_field='id'):
```

Comment:
Time to rename the function?

Reply:
`def geojson2jsonld(... `?

Reply:
lets change to `geojson2jsonld`

Reply:
updated this in api.py and linked_data.py